### ISSUE TITLE
PyQt recipe probably broken

### ISSUE THREAD

**Issue by extrowerk** [haikuports/haikuports#4380]
```
/bin/pyuic5: line 2: /packages/pyqt-5.13.1-1/cmd~python3.7/bin/python3: No such file or directory
```

The ```/packages/pyqt-5.13.1-1/``` folder have no ```cmd~python3.7/``` subfolder

**Comment by extrowerk** [comment]
This is x86_64, hrev53561, with pyqt_python3-5.13.1-1-x86_64.hpkg

**Comment by korli** [comment]
should be fixed in 49734d282de4ac269f34077da3b572a5e35eeb01
